src/BuildScriptGenerator/Ruby/RubyManifestFilePropertyKeys.cs

```diff
@@ -0,0 +1,14 @@
+// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+namespace Microsoft.Oryx.BuildScriptGenerator.Ruby
+{
+    internal static class RubyManifestFilePropertyKeys
+    {
+        internal const string RubyVersion = "rubyversion";
+
+        internal const string GemVersion = "gemversion";
```

Comment:
I would remove this since it comes as part of Ruby? and also make sure to remove it from `RubyScriptGeneratorOptions.cs` file